.gitignore

```diff
@@ -1,6 +1,6 @@
 *~
 local.mk
-build/
+/build/
```

Comment:
I was quite surprised to find that the new file was being ignored as well. Not sure if this is particular to some version of git or configuration but the proposed change fixes that, while keeping the base `build` directory ignored.

Reply:
> the new file

Which new file exactly? I don't really see why this change is necessary given that the only new file in this PR lives in the `external/` folder, so that should be completely separate from the `build/` directory.